modules/shutup/src/main/Dictionary.scala

```diff
@@ -33,13 +32,13 @@ bugger
 buk?kake
 bull?shit
 cancer
-cheat(ing|ed|er|s|)
+cheat(ing|ed|er|)
 chess(|-|_)bot(.?com)?
 chicken
 chink
 clit(oris|)
 clown
-cock
+cock(suc?ker|)
```

Comment:
You're missing a question mark at the end of the line

Reply:
The pipe is sufficient to work correctly. Maybe the question mark solution could be implemented instead for the whole dict at some point in the future.